releases/elasticsearch-exporter.yaml

```
@@ -30,13 +30,127 @@ releases:
+                Amazon ES stops taking automatic snapshots, even of healthy indices, while the red cluster status persists.
+          - alert: Elasticsearch_Cluster_Health_YELLOW
+            expr: elasticsearch_cluster_health_status{color="yellow"}==1
+            for: 300s
+            labels:
+              severity: warning
+            annotations:
+              summary:  "Elasticsearch cluster status is Yellow"
+              runbook_url: https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/aes-handling-errors.html#aes-handling-errors-yellow-cluster-status
+              description: >-
+                Elasticsearch cluster {{`{{ $labels.cluster }}`}}: not all replica shards are allocated to nodes.
+          - alert: Elasticsearch_Count_of_JVM_GC_Runs
+            expr: rate(elasticsearch_jvm_gc_collection_seconds_count{}[5m])>5
+            for: 60s
+            labels:
+              severity: "warning"
+            annotations:
+              summary:  "Elasticsearch: Count of JVM GC runs > 5 per sec"
+              description: >-
+                Elasticsearch node {{`{{ $labels.name }}`}}: Average of {{`{{ $value }}`}} JVM GC runs per
+                second over past 60 seconds.
+          - alert: Elasticsearch_GC_Run_Time
+            expr: rate(elasticsearch_jvm_gc_collection_seconds_sum[5m])>0.3
+            for: 60s
+            labels:
+              severity: "warning"
+            annotations:
+              summary:  "Elasticsearch GC running > 30% of real time"
+              description: >-
+                Elasticsearch node {{`{{ $labels.name }}`}} GC was running {{`{{ $value }}`}}% of real time over the past 60 seconds.
+          - alert: Elasticsearch_json_parse_failures
+            expr: increase(elasticsearch_cluster_health_json_parse_failures[10m])>0
+            for: 60s
+            labels:
+              severity: "warning"
+            annotations:
+              summary:  "Elasticsearch has json parse failures"
+              description: >-
+                Elasticsearch has had {{`{{ $value }}`}} json parse failures over the past 10 minutes
+          - alert: Elasticsearch_pending_tasks
+            expr: elasticsearch_cluster_health_number_of_pending_tasks > 0
+            for: 5m
+            labels:
+              severity: warning
+            annotations:
+              summary: "Elasticsearch has had pending tasks for more than 5 minutes"
+              description: "Elasticsearch pending tasks count of {{`{{ $value }}`}} is > 0 for 5 min. Cluster works slowly."
+          - alert: Elasticsearch_breakers_tripped
+            expr: increase(elasticsearch_breakers_tripped{}[5m])>0
+            for: 60s
+            labels:
+              severity: "warning"
+            annotations:
+              summary:  'Elasticsearch breaker{{`{{$labels.breaker}}`}} tripped'
```

```suggestion
              summary:  'Elasticsearch breaker {{`{{$labels.breaker}}`}} tripped'
```

do we need a space here?
we have it on line 154